.github/workflows/ci.yaml

```diff
@@ -52,7 +52,7 @@ jobs:
         uses: actions/cache@v2
         with:
           path: ${{ env.BUNDLE_PATH }}
-          key: gems-${{ matrix.os }}-${{ matrix.ruby }}-rails_61-${{ hashFiles('current_gemfile.lock') }}
+          key: gems-${{ matrix.os }}-${{ matrix.ruby.name }}-rails_61-${{ hashFiles('current_gemfile.lock') }}
```

Comment:
It's recommended that we don't cache the gems manually but instead allow the action to do so by specifying `bundler-cache: true` in the config. https://github.com/ruby/setup-ruby#caching-bundle-install-manually If we did that, then that would mean we could remove the following sections:
- Restore cached dependencies
- Cat current Gemfile to a specific file so caching works
- Install dependencies

Reply:
I think at some point I implemented these tricks because `ruby/setup-ruby` couldn't properly handle separate Gemfiles, but I'm pretty sure it handles them just fine now, we'll see :)

Reply:
That makes sense, thanks! 🙌🏻